bower_components/aura/lib/ext/mediator.js

```diff
@@ -92,7 +91,6 @@ define('aura/ext/mediator', function () {
        * @param {Object} payload    Payload emitted
        */
       app.sandbox.emit = function () {
-        if (this.stopped) { return; }
```

Comment:
I can say that it prevents this line from working: https://github.com/sulu-io/sulu/pull/1761/files#diff-54853adb238ed957cca321cfa254b7c1R101

I guess that's something that should be possible, since this sandbox and its corresponding component are still existing afterwards. Do you have any references where this might make trouble? Then we could check on these as well (and maybe find another solution).